### ISSUE TITLE
Glacier vault import fails

### ISSUE THREAD

**Issue by artamonovkirill** [hashicorp/terraform-provider-aws#12619]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

0.12.24

### Affected Resource(s)

* aws_glacier_vault

### Terraform Configuration Files

```hcl
resource "aws_glacier_vault" "photos" {
  name = "photos"
}
```

### Debug Output

https://gist.githubusercontent.com/artamonovkirill/528f00dd39c908586ed4e68223e2f7cc/raw/477a01e35563e5a27d0cc0c805f0cc430bde73a0/debug.out

### Panic Output

https://gist.githubusercontent.com/artamonovkirill/528f00dd39c908586ed4e68223e2f7cc/raw/477a01e35563e5a27d0cc0c805f0cc430bde73a0/crash.log

### Expected Behavior

Glacier vault is imported.

### Actual Behavior

Terraform crashed.

### Steps to Reproduce

`terraform import aws_glacier_vault.photos photos`

### Important Factoids

No.

### References

n/a

**Comment by ewbankkit** [comment]
```
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: 2020/04/01 22:03:30 [DEBUG] [aws-sdk-go] DEBUG: Response glacier/GetVaultAccessPolicy Details:
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: ---[ RESPONSE ]--------------------------------------
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: HTTP/1.1 403 Forbidden
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: Connection: close
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: Content-Length: 225
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: Content-Type: application/json
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: Date: Wed, 01 Apr 2020 20:03:29 GMT
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: X-Amzn-Requestid: v_XUea471eAsf4s-CKDM8LOOOeXEzzHzVBjYpjwma-oGA-s
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: 
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: 
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: -----------------------------------------------------
2020-04-01T22:03:30.382+0200 [DEBUG] plugin.terraform-provider-aws_v2.55.0_x4: 2020/04/01 22:03:30 [DEBUG] [aws-sdk-go] {"code":"AccessDeniedException","type":"Client","message":"User: arn:aws:iam::123456789012:user/me is not authorized to perform: glacier:GetVaultAccessPolicy on resource: arn:aws:glacier:eu-west-1:123456789012:vaults/photos"}
```

https://github.com/terraform-providers/terraform-provider-aws/blob/c5e26ae4e683f5eee4724acb25355f665cb5e1ff/aws/resource_aws_glacier_vault.go#L172-L182

`pol != nil` but `*pol.Policy == nil`.
The error handling here looks faulty.

**Comment by artamonovkirill** [comment]
I still get a panic with 2.60.0, but now a different error: [crash.out](https://gist.githubusercontent.com/artamonovkirill/528f00dd39c908586ed4e68223e2f7cc/raw/b547b63a9102581574d0abb699f823239713bbc6/crash.log) and [debug.out](https://gist.githubusercontent.com/artamonovkirill/528f00dd39c908586ed4e68223e2f7cc/raw/b547b63a9102581574d0abb699f823239713bbc6/debug.out)

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.27.0 of the Terraform AWS Provider, next week. Thank you to @DrFaust92 for the implementation. 👍
